_data/collaborators.yml

```
@@ -35,3 +35,14 @@
       link: https://www.linkedin.com/in/nehha/
     - name: github
       link: https://github.com/Neha
+
+
+- name: Sophy O.
+  image: assets/images/collaborators/10.jpeg
+  position: UI Developer/UX Designer
+  comment: Team Collaborator
```

```suggestion
  comment: Website Collaborator
```

As a suggestion, you can add the topic you would be interested in collaborate, for example now it is the website.

But if you would like to collaborate with anything, please keep as `Team Collaborator`